### PR TITLE
Make changes to tooltip styles

### LINKED ISSUE #753: Small changes to tooltips
@issa-tseng writes on Slack:

> - instant tooltips are nice but i'd eliminate the fade.
>   - it's also a little bottom heavy? like either a px or two needs to be added to the top or taken from the bottom or something between but there's more space at the bottom than the top rn

### PR DESCRIPTION
**Pull request by matthew-white**
Closes #753.

#### What has been done to verify that this works as intended?

I viewed the results locally. Once the PR is on staging, I'll ask issa to take a look.

#### Why is this the best possible solution? Were any other approaches considered?

I'm going to leave comments about individual changes I made.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

The risk of regression should be low.

#### Does this change require updates to user documentation? If so, please file an issue [here](https://github.com/getodk/docs/issues/new) and include the link below.

No changes needed.

#### Before submitting this PR, please make sure you have:

- [x] run `npm run test` and `npm run lint` and confirmed all checks still pass OR confirm CircleCI build passes
- [x] verified that any code or assets from external sources are properly credited in comments or that everything is internally sourced

### CODE REVIEW

File: src/assets/scss/app.scss [hunk 1]
```diff
@@ -86,7 +86,9 @@ em:lang(ja) {
   left: -1000px;
 }
 .tooltip-inner {
+  border-radius: 2px;
```
Comment (matthew-white) [inline src/assets/scss/app.scss:89]: The Bootstrap default is 4px. 2px matches the `border-radius` of `.btn`.

File: src/assets/scss/app.scss [hunk 2]
```diff
@@ -86,7 +86,9 @@ em:lang(ja) {
...
   overflow-wrap: break-word;
+  padding-bottom: 2px;
```
Comment (matthew-white) [inline src/assets/scss/app.scss:91]: The Bootstrap default is 3px, so this is a reduction of 1px.

issa wrote:

> either a px or two needs to be added to the top or taken from the bottom or something between

I tried adding 1px to the top, but even that felt like possibly too much padding. I think it's nice for tooltips to be smaller and to not have too much padding.

I also tried removing 2px from the bottom, but that felt like too much.

File: src/directives/tooltip.js [hunk 3]
```diff
@@ -265,7 +265,7 @@ class Tooltip {
     }
 
     this.position()
-      .then(() => { this.element.classList.add('in', 'fade'); })
+      .then(() => { this.element.classList.add('in'); })
```
Comment (matthew-white) [inline src/directives/tooltip.js:268]: We need to keep the `in` class because it sets the tooltip's opacity.

**Review by sadiqkhoja — APPROVED** [review]
Code looks good. I wish there was a screenshot in the PR as well

**Pull request merged**